specification/eventgrid/data-plane/Microsoft.AgFoodPlatform/stable/2018-01-01/AzureDataManagerForAgriculture.json

```diff
@@ -0,0 +1,2245 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Schema of Azure Data Manager for Agriculture events published to Azure Event Grid",
+    "description": "Describes the schema of the Azure Data Manager for Agriculture Configuration events published to Azure Event Grid. This corresponds to the Data property of an EventGridEvent.",
+    "version": "2018-01-01"
+  },
+  "paths": {},
+  "definitions": {
+    "AttachmentResourceType": {
+      "description": "Attachment resource type.",
+      "enum": [
+        "Party",
+        "Farm",
+        "Field",
+        "SeasonalField",
+        "Boundary",
+        "ApplicationData",
+        "HarvestData",
+        "TillageData",
+        "PlantingData",
+        "PlantTissueAnalysis"
+      ],
+      "type": "string",
+      "x-ms-enum": {
+        "name": "AttachmentResourceType",
+        "modelAsString": true
+      }
+    },
+    "BoundaryParentType": {
+      "description": "Boundary parent type enum.",
+      "enum": [
+        "Field",
+        "SeasonalField",
+        "Zone",
+        "Prescription",
+        "PlantTissueAnalysis",
+        "ApplicationData",
+        "PlantingData",
+        "TillageData",
+        "HarvestData"
+      ],
+      "type": "string",
+      "x-ms-enum": {
+        "name": "BoundaryParentType",
+        "modelAsString": true
+      }
+    },
+    "DataManagerForAgriApplicationDataChangedV2EventData": {
```

Comment:
@JoshLove-msft  please let us know if you are okay with `DataManagerForAgri` prefix for each event schema class name. 

FYI. our service name is `Microsoft Azure Data Manager for Agriculture` and we selected shorten version of it.

Reply:
My hunch is that this is not okay and we would want the full `Agriculture` spelled out but this is something we can discuss at the arch board meeting on 4/5.

Reply:
AgriFoodFarming? Drop DataManagerFor. Will discuss offline.

Reply:
Suggested Prefix for all definitions is AgriFoodFarming.

Reply:
From @bhanuprakash-1:
> Hi [@JoshLove-msft](https://github.com/JoshLove-msft) , we strongly want to use the prefix "DataManagerForAgri" or "DataManagerForAgriculture"(Bit lengthy, worried might become issue in some model names etc in SDK).
 "data-manager-for-agri" is our marketing slug value for our Azure service. 
Also, the "DataManagerForXYZ" is the pattern for our org(MCIGET) and so there will be other services in future with similar naming conventions for other industries.
For reference: https://learn.microsoft.com/en-us/rest/api/data-manager-for-agri/
https://learn.microsoft.com/en-us/azure/data-manager-for-agri/overview-azure-data-manager-for-agriculture

Reply:
Despite the fact that we use data-manager in marketing materials, we don't use marketing info in our type names as those are subject to change over time. 
The recommendation is to use AgriFoodFarming as it best align with other shipped AgriFood libraries and is what the architects feel comfortable shipping in the Event Grid client libraries.